lib/school.rb

```diff
@@ -0,0 +1,8 @@
+class School
+  attr_accessor :start_time, :hours_in_day
```

Comment:
Hey have you thought about using this as two lines?

Reply:
Have you thought about using an attr_reader?  We don't want people to change the attribute outside of the class.

Reply:
Thanks for catching those! I'll make the changes right away 🖖